neurobooth_os/gui.py

```diff
@@ -650,7 +646,12 @@ def main():
         help="Specify which database to connect",
     )
     (options, args) = parser.parse_args()
-    gui(remote=options.remote, database=options.database)
+    try:
+        gui(remote=options.remote, database=options.database)
+    except Exception as e:
+        logger.critical(f"An uncaught exception occurred. Exiting: {repr(e)}")
+        logger.critical(e, exc_info=sys.exc_info())
```

Comment:
If I understand correctly, exception calls error(). The difference between error and critical (according to the internet are: 

"Error = 40: As it implies, an error has occurred. The software was unable to perform some function.
Critical = 50: A serious error has occurred. The program itself may shut down or not be able to continue running properly."

Since these exceptions will result in a shutdown, i went with critical.

Reply:
It may be good to do this for the outer exception cause in the dump script too then. I was also wondering if we get more information with `exc_info=sys.exc_info()`, or if `.exception` is simply doing that argument implicitly?

Reply:
I believe ex_info=sys.exc_info() is necessary for the stack trace to be included

Reply:
I will fix the dump exception handling.